.release-notes/3657.md

```diff
@@ -0,0 +1,14 @@
+## Restrict Array.chop to be used with sendable element types
```

Comment:
Can you change this to be `Fix soundness problem with Array.chop`

It's a much clearer end user "what was the issue" headline.

Reply:
Sure thing, changed